src/test/java/com/wikia/webdriver/testcases/mobilewikitests/ArticlePageTests.java

```diff
@@ -133,28 +130,28 @@ public void mercury_article_navigateToArticlesWithColonAndQuestionMark() {
 
     Log.warning("Info", "Accessing article through link in navigation side");
 
-    topBar.openNavigation();
+    globalNavigationMobile.openNavigation();
     //navigation.openSubMenu(3);
-    navigation.openPageLink(5);
+    //navigation.openPageLink(5);
 
     Assertion.assertFalse(driver.getCurrentUrl().contains(encodedColonUrl));
     Assertion.assertTrue(MercurySubpages.COLON.toLowerCase().contains(article.getArticleTitle().toLowerCase()));
 
-    topBar.openNavigation();
+    globalNavigationMobile.openNavigation();
     //navigation.openSubMenu(3);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ACommentedOutCodeLine)